native/effektio/api.rsh

```diff
@@ -138,6 +177,19 @@ object Conversation {
     /// returns the event_id as given by the server of the event soon after
     /// received over timeline().next()
     fn send_plain_message(text_message: string) -> Future<Result<string>>;
+
+    fn send_image_message(uri: string, name: string, mimetype: string, size: u32, width: u32, height: u32) -> Future<Result<string>>;
+
+    /// decrypted image file data
+    fn image_binary(event_id: string) -> Future<Result<buffer<u8>>>;
```

Comment:
why is this function not on `image` or `RoomMessage`?

Reply:
Flutter keeps room as member variable in UI.
Flutter doesn't keep room message as variable, because it may have to load so many messages in screen.
So I wanted to get image binary data from member variable.
Getting temporary variable for each message may be slow.

Reply:
yeah, fair enough then. keep it the way it is.